src/components/pressables/IconButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import cn from 'classnames'
+import { Pressable } from 'react-native'
+import { IconButtonProps } from '@/types/iconButtton'
+import Icon from '../Icon/Icon'
+
+export const IconButton = (props:IconButtonProps): JSX.Element => {
+  const {
+    size,
+    colour,
+    bgClass = '',
+    icon,
+    disabled = false,
+    onPress,
+  } = props
+
+  const buttonBackground = bgClass.length === 0 ? 'bg-indigo-300' : bgClass
+
+  const buttonStyles = cn({
+    'flex items-center justify-center rounded-full': true,
+    'h-8 w-8': size === 'sm',
```

Comment:
Can we use the IconSize enum here too please?

Reply:
👍